pkg/internalregistry/controllers/service_account_controller.go

```diff
@@ -117,6 +119,8 @@ func NewServiceAccountController(kubeclient kubernetes.Interface, serviceAccount
 
 const serviceAccountControllerFieldManager = "openshift.io/image-registry-pull-secrets_service-account-controller"
 
+var imagePullSecretFieldPath, _ = fieldpath.MakePath("imagePullSecrets")
```

Comment:
No longer needed in the latest revision?

Reply:
Removed.